src/main/java/King.java

```diff
@@ -0,0 +1,173 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class King {
+    public static final String name = "King";
+    public static int repeatCount = 0;
+    public static int tasksCount = 0;
+    public static ArrayList<Task> tasks = new ArrayList<>();
+
+    public static void greet() {
+        String logo =
+                "| |/ /|_ _|| \\| | / _` |\n" +
+                "| ' <  | | | .` || (_| |\n" +
+                "|_|\\_\\|___||_|\\_| \\__,_|\n";
+
+        System.out.println("Hello from\n" + logo);
+
+        System.out.println(" Hello! I'm " + name + "\n" +
+                           " What can I do for you?\n" +
+                           "____________________________________________________________\n");
```

Comment:
Could you save the "______...______" portion of the output as it's own string that you can easily call whenever you need it?